src/witan/models/dem/ccm/fert/fertility.clj

```diff
@@ -104,28 +103,74 @@
          (calc-scaled-fert-rates asfr-birth-data-year)
          (ds/select-columns [:gss-code :sex :age :year :fert-rate]))}))
 
-(defworkflowfn project-asfr-finalyearhist-fixed
-  "Takes dataset of historic age specific fertility rates, and parameter
-   for the base year of fertility data. Returns dataset with projected
-   age specific fertility rates, calculated using the jumpoff year average
-   method (see docs)."
-  {:witan/name :ccm-fert/project-asfr-finalyearhist-fixed
+(defworkflowfn project-asfr-1-0-0
+  "Project ASFR finalyearhist fixed. Takes dataset of historic age specific 
+   fertility rates, and parameter for the base year of fertility data. 
+   Returns dataset with projected age specific fertility rates, calculated 
+   using the jumpoff year finalyearhist method (see docs)."
+  {:witan/name :ccm-fert/project-asfr
    :witan/version "1.0.0"
    :witan/input-schema {:historic-asfr HistASFRSchema}
    :witan/output-schema {:initial-projected-fertility-rates ProjFixedASFRSchema}
    :witan/exported? true}
   [{:keys [historic-asfr]} _]
-  (let [final-year (reduce max (ds/column historic-asfr :year))
+  (let [final-year (m-utils/get-last-year historic-asfr)
         _ (utils/property-holds? final-year m-utils/year? (str final-year " is not a year"))
         final-year-hist (wds/select-from-ds historic-asfr {:year final-year})]
     {:initial-projected-fertility-rates
      (ds/select-columns final-year-hist [:gss-code :sex :age :fert-rate])}))
 
-(defworkflowfn project-births-from-fixed-rates
-  "Takes a dataset with population at risk from the current year of the projection
-  loop and another dataset with fixed fertility rates for the population. Returns a
-  dataset with a column of births, which are the product of popn at risk & the rates"
-  {:witan/name :ccm-fert/project-births-fixed-rates
+(defn project-asfr
```

Comment:
Can this have a better name? Confusing now we have `project-asfr-1-0-0` etc. Maybe `project-asfr-internal` ?